contained-innersource-enables-collaborative-product-development.md

```diff
@@ -0,0 +1,35 @@
+## Title  
+Contained inner source enables collaborative product development
+
+## Context
+* Multiple business units need a software component.
+* It makes sense for the business units to collaborate; each needs some different changes to the code.
```

Comment:
Maybe write that the BUs _are in agreement_ that it makes sense to collaborate. This would be more clear to me.